src/main/groovy/au/org/ala/ecodata/metadata/OutputMetadata.groovy

```diff
@@ -165,7 +165,7 @@ class OutputMetadata {
         dataModelIterator { path, node ->
             if (!visitedDataNodes.contains(node)) {
                 println "Warning: node ${path} is not refernced by the view model"
-                callback(fullPathToNode(path, node), null, node)
+                callback(path, null, node)
```

Comment:
@chrisala is this code going to negatively impact MERIT ?

Reply:
It will certainly impact MERIT, I'm hoping it will do so in a good way.  MERIT doesn't use editable fields in footers so I'm expecting the only change will be where there is an error in the form such that the data model isn't attached to the view in which case the value will be null in both cases.  It will only impact the downloads anyway so I'm not too worried about it.